idea/src/org/jetbrains/kotlin/idea/inspections/ReplaceWithOperatorAssignmentInspection.kt

```diff
@@ -55,7 +55,7 @@ class ReplaceWithOperatorAssignmentInspection : AbstractApplicabilityBasedInspec
     override val defaultFixText = "Replace with operator-assignment"
 
     override fun fixText(element: KtBinaryExpression) =
-            "Replace with '${element.operationReference.operationSignTokenType?.value}='"
+        "Replace with '${(element.right as KtBinaryExpression).operationReference.operationSignTokenType?.value}='"
```

Comment:
Just for safety, I'd use `as?` instead of `as` here

Reply:
Also, some test should be provided. Please look at `idea/testData/inspectionsLocal/replaceWithOperatorAssignment` for examples. You should use "Generate All Tests" run configuration to regenerate automatic tests (`LocalInspectionTestGenerated` in our case).

Reply:
Also I recommend to run other local inspection tests to be sure that nothing is broken